graylog2-web-interface/src/components/lookup-tables/DataAdapterForm.tsx

```diff
@@ -274,143 +111,175 @@ class DataAdapterForm extends React.Component<
     return <span>{defaultText}</span>;
   };
 
-  // eslint-disable-next-line
-  _renderTitle = (title, typeName, create) => {
-    const TagName = create ? 'h3' : 'h2';
+  const DocComponent = React.useMemo(() => plugin.documentationComponent, [plugin]);
+  const pluginDisplayName = React.useMemo(() => plugin.displayName || type, [plugin, type]);
 
-    return (
-      <TagName>
-        {title} <small>({typeName})</small>
-      </TagName>
-    );
-  };
+  const sanitizeName = (inName: string) => inName.trim().replace(/\W+/g, '-').toLocaleLowerCase();
 
-  render() {
-    const { dataAdapter, isFormDisabled } = this.state;
-    const { create, type, title, history } = this.props;
-    const adapterPlugins = PluginStore.exports('lookupTableAdapters');
+  const handleTitleChange = (values: LookupTableAdapter, setValues: any) => (event: React.BaseSyntheticEvent) => {
+    if (!generateName) return;
+    const safeName = sanitizeName(event.target.value);
 
-    const plugin = adapterPlugins.filter((p) => p.type === type);
-    const onCancel = () => history.push(Routes.SYSTEM.LOOKUPTABLES.DATA_ADAPTERS.OVERVIEW);
-    let configFieldSet = null;
-    let documentationComponent = null;
-    let pluginDisplayName = dataAdapter.config.type;
+    setValues({
+      ...values,
+      title: event.target.value,
+      name: safeName,
+    });
+  };
 
-    if (plugin && plugin.length > 0) {
-      const p = plugin[0];
+  const updateCustomErrorTTL = (value: number, enabled: boolean, unit: string, values: LookupTableAdapter, setValues: any) => {
+    setValues({
+      ...values,
+      custom_error_ttl: value,
+      custom_error_ttl_enabled: enabled,
+      custom_error_ttl_unit: unit,
+    });
+  }
 
-      pluginDisplayName = p.displayName;
+  const handleValidation = (values: LookupTableAdapter) => {
+    const errors: any = {};
 
-      configFieldSet = React.createElement(p.formComponent, {
-        config: dataAdapter.config,
-        handleFormEvent: this._onConfigChange,
-        updateConfig: this._updateConfig,
-        validationMessage: this._validationMessage,
-        validationState: this._validationState,
-        setDisableFormSubmission: this._setIsFormDisabled,
-      });
+    if (!values.title) errors.title = 'Required';
 
-      if (p.documentationComponent) {
-        documentationComponent = React.createElement(p.documentationComponent, {
-          dataAdapterId: dataAdapter.id,
-        });
-      }
+    if (!values.name) {
+      errors.name = 'Required';
+    } else {
+      validate(values);
     }
 
-    let documentationColumn = null;
-    let formRowWidth = 8; // If there is no documentation component, we don't use the complete page
+    if (values.config?.type !== 'none' && configReady) {
+      const confErrors = configRef.current?.validate?.() || {};
```

Comment:
Getting an error if I click into and out of any field on the `[Deprecated] Ransomware blocklists from abuse.ch` data adapter:
<img width="1212" height="1022" alt="image" src="https://github.com/user-attachments/assets/ea6ad757-aa10-4327-9f55-bc3b106689d7" />

Reply:
Hey @ryan-carroll-graylog this should be fixed with the merge of the lookup table modals